_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -2962,6 +2962,22 @@ mfxStatus VideoVPPHW::Close()
 } // mfxStatus VideoVPPHW::Close()
 
 
+bool VideoVPPHW::UseCopyPassThrough() const
+{
+    if (m_pCore->GetVAType() != MFX_HW_VAAPI || m_ioMode != D3D_TO_D3D)
+    {
+        return true; // preferred unconditionally
```

Comment:
after function rename this comment is misleading

Reply:
Done.